### ISSUE TITLE
ERROR: One line install script doesn't work well.

### ISSUE THREAD

**Issue by indrakaw** [Szero/volaupload.sh#4]
I tried this (as written on README.md):
```
curl -sLo- https://rawgit.com/Szero/volascripts.sh/master/install.sh | bash
```
This is what I got:
![2018-01-08-164457_1366x768_scrot](https://user-images.githubusercontent.com/9590878/34665636-942cc076-f493-11e7-8073-32e67750244b.png)

Try ahead in clean machine.

As alternative, I downloaded `install.sh` locally then run it.
```
wget https://rawgit.com/Szero/volascripts.sh/master/install.sh
bash ./install.sh
```
No error indeed. :thinking:

**Comment by laino** [comment]
the first one doesn't get its input from console STDIN, so the prompt doesn't work.

**Comment by szero** [comment]
@laino  Thanks for noticing that! I tested this script locally too, so it ended up busted in the end. Process substitution seems to resolve this.
@indrakaw Try using this command instead: 
```bash
bash <(curl -sLo- https://rawgit.com/Szero/volascripts.sh/master/install.sh)
```

**Comment by szero** [comment]
Fixed in ca24d41
